docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -121,6 +132,55 @@ async function renderApiDocumentation() {
 	console.groupEnd();
 }
 
+/**
+ * Processes documents and generates data required for the nav bar.
+ * @param {Array<Object>} documents - List of documents with apiItem.
+ * @param {ApiItem | undefined} documents.apiItem - The API item that the document is created from. Some documents may not have an apiItem.
+ */
+function buildNavBar(documents) {
```

Comment:
Nit: how would you feel about moving this to a separate file in this directory? Among other things, it would make it easier to add unit tests for the logic in the future (which I think we should start doing here at some point).